docs/api/web-contents.md

```diff
@@ -647,7 +647,8 @@ Removes the specified path from DevTools workspace.
 ### `webContents.openDevTools([options])`
 
 * `options` Object (optional)
-  * `detach` Boolean - opens DevTools in a new window
+  * `mode` String - Opens the devtools with specified dock state, can be one of
+    "right", "bottom", "undocked", "detach". Defaults to last used dock state.
```

Comment:
What's the difference between `undocked` and `detach`? They seem to do the same thing. If there is a difference, it should be documented.